themes/navy/layout/partial/after_footer.swig

```diff
@@ -1,14 +1,14 @@
 <!-- Scripts -->
 <!-- Cookie -->
-<script src="https://cdnjs.cloudflare.com/ajax/libs/js-cookie/2.2.0/js.cookie.min.js"></script>
+<script src="https://cdn.jsdelivr.net/npm/js-cookie@2.2.0"></script>
```

Comment:
prefer "https://cdn.jsdelivr.net/npm/js-cookie@2.2.0/src/js.cookie.min.js" which clearly show which file is used.

Reply:
Since the [README of `js-cookie`](https://github.com/js-cookie/js-cookie/#installation) recommends using `https://cdn.jsdelivr.net/npm/js-cookie@2/src/js.cookie.min.js`, I will update to that.